libvast/vast/fbs/partition.fbs

```diff
@@ -0,0 +1,34 @@
+namespace vast.fbs;
+
+table ValueIndex {
+  /// The full-qualified field name, e.g., "zeek.conn.id.orig_h".
+  qualified_field_name: string;
```

Comment:
```suggestion
  qualified_field_name: string (key);
```

I don't recall if the value indexes are sorted, but  if they are, this gives *O(log n)* access.

Reply:
They're not, the order is determined by the order of the `combined_layout`, which is determined by the order in which we saw the table columns.